docs/user_guide/emummc/sd_preparation.md

```diff
@@ -18,23 +18,14 @@ Atmosphere has its own bootloader, called fusee (primary). For the purposes of t
 
 !!! tip ""
     - The latest release of <a href="https://github.com/CTCaer/Hekate/releases/" target="_blank">Hekate</a> (Download the `hekate_ctcaer_(version).zip` release of hekate)
-  
     - The hekate config file: <a href="../../../files/emu/hekate_ipl.ini" download>hekate_ipl.ini</a>
-  
     - The bootlogo zip folder: <a href="../../../files/bootlogos.zip" download>bootlogos.zip</a>
-  
-    - The latest release of <a href="https://github.com/Atmosphere-NX/Atmosphere/releases" target="_blank">Atmosphere</a>  (Download the `atmosphere-(version).zip` release of Atmosphere.)
-  
-     - The latest release of <a href="https://github.com/shchmue/Lockpick_RCM/releases" target="_blank">Lockpick_RCM</a>  (Download the `Lockpick_RCM.bin` release of Lockpick)
-  
-    - The latest release of <a href="https://github.com/FlagBrew/Checkpoint/releases" target="_blank">Checkpoint</a>  (Download the `Checkpoint.nro` release of Checkpoint)
-  
-    - The latest release of <a href="https://github.com/mtheall/ftpd/releases" target="_blank">FTPD</a>  (Download the `ftpd.nro` release of FTPD)
-  
-    - The latest release of <a href="https://github.com/exelix11/SwitchThemeInjector/releases" target="_blank">NXThemeInstaller</a>  (Download the `NxThemesInstaller.nro` release of NxThemeInstaller)
-  
+    - The latest release of <a href="https://github.com/Atmosphere-NX/Atmosphere/releases" target="_blank">Atmosphere</a> (Download the `atmosphere-(version).zip` release of Atmosphere.)
+     - The latest release of <a href="https://github.com/shchmue/Lockpick_RCM/releases" target="_blank">Lockpick_RCM</a> (Download the `Lockpick_RCM.bin` release of Lockpick)
+    - The latest release of <a href="https://github.com/FlagBrew/Checkpoint/releases" target="_blank">Checkpoint</a> (Download the `Checkpoint.nro` release of Checkpoint)
+    - The latest release of <a href="https://github.com/mtheall/ftpd/releases" target="_blank">FTPD</a> (Download the `ftpd.nro` release of FTPD)
+    - The latest release of <a href="https://github.com/exelix11/SwitchThemeInjector/releases" target="_blank">NXThemeInstaller</a> (Download the `NxThemesInstaller.nro` release of NxThemeInstaller)
```

Comment:
Only a teeny-tiny change: But maybe we coule change `NxThemeInstaller` to `NxThemesInstaller` to stay consistent